inmemory/kv.go

```diff
@@ -136,33 +136,16 @@ func put(shardDB *ShardDB, bucket string, key, value []byte, ttl uint32, flag ui
 	valueSize := uint32(len(value))
 	timestamp := uint64(time.Now().Unix())
 	bucketSize := uint32(len(bucket))
+	meta := nutsdb.NewMetaData().WithTimeStamp(timestamp).WithKeySize(keySize).WithValueSize(valueSize).WithFlag(flag).WithTTL(ttl).
+		WithBucketSize(bucketSize).WithStatus(nutsdb.Committed).WithDs(nutsdb.DataStructureBPTree)
 	err = shardDB.BPTreeIdx[bucket].Insert(key, &nutsdb.Entry{
```

Comment:
Seems this line can refactor to the new initial way of Entry, could you help to do me a favor?